index.bs

```diff
@@ -122,7 +117,7 @@ spec: ECMAScript; urlPrefix: https://tc39.github.io/ecma262/#
 spec: RFC8152; urlPrefix: https://tools.ietf.org/html/rfc8152
     type: dfn
         text: Section 7; url: section-7
-
+<!--
```

Comment:
Why not delete the obsolete reference?

Reply:
oh sorry, fergot it was still lying about...   deleted.